packages/backend-core/src/db/conversions.js

```diff
@@ -0,0 +1,46 @@
+const NO_APP_ERROR = "No app provided"
+const { APP_DEV_PREFIX, APP_PREFIX } = require("./constants")
+
+exports.isDevAppID = appId => {
+  if (!appId) {
+    throw NO_APP_ERROR
+  }
+  return appId.startsWith(APP_DEV_PREFIX)
+}
+
+exports.isProdAppID = appId => {
```

Comment:
While we're touching this stuff, could we either make this `isDeployedAppID` or update the `getDeployedAppID` to be `getProdAppID`. Any preference here? kind of annoying that we use deployed and production interchangeably

Reply:
yeah you're right, may as well update it now, think we should move to `prod` term, as deployment/deployed isn't used much really.